src/main/java/org/isf/admission/gui/AdmissionBrowser.java

```diff
@@ -1826,8 +1826,6 @@ private JButton getSaveButton() {
 					admission.setType("N");
 				}
 
-				admission.setDeleted("N");
-
```

Comment:
maybe this change comes from issue OP-780 that is still in review, so this piece of code is still needed